docs/testcases/UI/Run_tools_in_sandbox/Limit_mounts/2354/2354_2.md

```diff
@@ -0,0 +1,41 @@
+# Check disable mounting of specific data storage for Pipeline and Detached Configuration
+
+
+Test verifies that
+- object storage mount can be disabled at object storage editing
+- object storage disabled to mount is hidden from the Limit mounts dialogs for the Pipeline and Detached Configuration
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+
+**Preparations**:
+1. Login as the admin user from the prerequisites 
+2. Create *object storage*
+3. Create *pipeline*
+4. Create *detached configuration*
+5. Give all permissions on *pipeline* and *detached configuration* to the non-admin user from the prerequisites
```

Comment:
I suggest to give at least `read` permission to *object storage* as well. Otherwise, *object storage* will be invisible at steps 14, 18 by another reason (not the "Disable mount" option)

Reply:
Done